spec/system/new_topic_list_spec.rb

```diff
@@ -102,7 +104,7 @@
         expect(tabs_toggle.replies_tab).to have_count(3)
         expect(tabs_toggle.topics_tab).to have_count(3)
 
-        expect(current_url).to end_with("/new?subset=topics")
```

Comment:
@OsamaSayegh Do note that asserting the for current url this way does not tell capybara to wait until the current url changes to what we expect. Note that `current_url` simply returns the url of the browser at the point the URL is executed. However, if you use `have_current_path`, Capybara will actually retry until the expected URL is present or wait until the default wait timeout is exhausted.